CONTRIBUTING.md

```diff
@@ -1,9 +1,8 @@
 ### Environment setup
 
 1. (macOS only) Install Xcode Command Line Tools: `xcode-select --install`
-2. Install Node.js
-4. Bootstrap the packages: `npm run bootstrap`
-5. (Optional, macOS and Linux) Install the [status bar indicator](https://github.com/roman01la/anybar-webpack#known-apps) app to receive webpack build notifications.
+2. Install Node.js@16 (with NPM@8). We suggest using NVM for Node version management
```

Comment:
I find `n` more convenient

Reply:
@Hypnosphi yep that makes sense. Let's suggest to use any of these